ts/components/search/SearchButton.tsx

```diff
@@ -50,25 +47,21 @@ class SearchButton extends React.Component<Props, State> {
     return (
       <React.Fragment>
         {searchText.isSome() ? (
-          <Item>
-            <NavigationEvents onWillBlur={this.onSearchDisable} />
-            <Input
-              placeholder={I18n.t("global.actions.search")}
-              value={searchText.value}
-              onChangeText={this.onSearchTextChange}
-              autoFocus={true}
-              placeholderTextColor={color(variables.brandGray)
-                .darken(0.2)
-                .string()}
-            />
-            <ButtonDefaultOpacity
-              onPress={this.onSearchDisable}
-              transparent={true}
-              accessibilityLabel={I18n.t("global.buttons.close")}
-            >
-              <IconFont name="io-close" color={this.props.color} />
-            </ButtonDefaultOpacity>
-          </Item>
+          <LabelledItem
```

Comment:
It has a wrong behaviour when it's used in `MESSAGE_HOME`.
| Before | After|
|-------|-------|
|<img src="https://user-images.githubusercontent.com/11773070/126488582-cf9c46d7-7c0f-476c-be91-123efdbbd26e.png" width=200>|<img src="https://user-images.githubusercontent.com/11773070/126488539-4de4755a-4cce-4b4f-a335-4eea226253c7.png" width=200>|

Reply:
ok, this clearly needs a fix :)